cmd/compatibility.go

```diff
@@ -0,0 +1,32 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/buildpacks/lifecycle/api"
+)
+
+func VerifyCompatibility() error {
```

Comment:
I don't feel strongly but this could probably move into `version.go` where PlatformAPI is defined